modyn/selector/internal/selector_strategies/abstract_selection_strategy.py

```diff
@@ -217,59 +215,41 @@ def trigger(self) -> tuple[int, int, int, dict[str, Any]]:
         partition: Optional[int] = None
         swt.start("on_trigger")
 
-        for partition, (training_samples, partition_log) in enumerate(self._on_trigger()):
+        for partition, (training_samples, partition_log) in enumerate(training_set_producer()):
             overall_partition_log = {"partition_log": partition_log, "on_trigger_time": swt.stop("on_trigger")}
 
-            logger.info(
-                f"Strategy for pipeline {self._pipeline_id} returned batch of"
-                + f" {len(training_samples)} samples for new trigger {trigger_id}."
-            )
-
```

Comment:
removed because I think it's not very useful, and if we had to keep it, I have to add another parameter to the function `source_pipeline_id`, because now from which pipeline the data comes from can be different than which pipeline it goes to.

I just don't want to add another parameter simply to keep a logging message.